NEWS.md

```diff
@@ -1,4 +1,60 @@
-afew 3.1.0 (unreleased)
+afew n.n.n (unreleased)
+=======================
+
```

Comment:
If you look further down in the file, these were usually a bit of a summary, not just a dump of `git log`, which is easier for a user to create on their own. 

In any case, it's unrelated to this PR, so should be in a separate one.

Reply:
Ok I'll put that in a seprate PR